test/setup.spec.js

```diff
@@ -20,6 +20,28 @@ describe('setup', () => {
     expect(vm.a).toBe(1);
   });
 
+  it('should work with non reactive null', () => {
+    const vm = new Vue({
+      setup() {
+        return {
+          a: null,
+        };
+      },
+    }).$mount();
+    expect(vm.a).toBe(null);
+  });
+  
+  it('should work with non reactive undefined', () => {
+    const vm = new Vue({
+      setup() {
+        return {
+          a: undefined,
+        };
+      },
+    }).$mount();
+    expect(vm.a).toBe(undefined);
```

Comment:
Not a valid test case. Because the `vm.a` would also be `undefined` even when the `setup` function throws.
The alternative is to either test against the log message for warning messages or add an extra non-empty field and see if the value also successfully returns (I personally prefer the latter, as it does not depend on internal implementations).